### PR TITLE
CQ-233: Add human-readable filters description 

### PR DESCRIPTION
**Pull request by torchbear**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

File: gooddata-sdk/gooddata_sdk/compute/model/filter.py [hunk 1]
```diff
@@ -18,6 +19,17 @@
 from gooddata_sdk.compute.model.base import Filter, ObjId
 from gooddata_sdk.compute.model.metric import Metric
 
+_DATE_FORMAT_INPUT = "%Y-%m-%d"
+_DATE_FORMAT_OUTPUT = "%-m/%-d/%Y"
```
Comment (no23reason) [inline gooddata-sdk/gooddata_sdk/compute/model/filter.py:23]: What do the `%-m` and `%-d` placeholders mean? I don't see them in the docs: https://docs.python.org/3/library/datetime.html#strftime-and-strptime-format-codes

Comment (torchbear) [reply gooddata-sdk/gooddata_sdk/compute/model/filter.py:23]: it's documented even deeper :D 
https://manpages.debian.org/bookworm/manpages-dev/strftime.3.en.html#glibc_notes

**Review by no23reason — APPROVED** [review]
LGTM

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-python-sdk-gate-PR579

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-fossa-scanning-zuul/330/ : FAILURE in 1m 20s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-sonarqube-scan-zuul-gate/480/ : SUCCESS in 1m 31s
- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-merge-compliance-check/484/ : SUCCESS in 11s

**Pull request merged**